frontend/public/components/app-contents.tsx

```diff
@@ -532,78 +532,6 @@ const AppContents: React.FC<{}> = () => {
         }
       />
 
-      <LazyRoute
-        path="/monitoring/alerts"
-        exact
-        loader={() =>
-          import('./monitoring/alerting' /* webpackChunkName: "alerting" */).then(
-            (m) => m.MonitoringUI,
-          )
-        }
-      />
-      <LazyRoute
-        path="/monitoring/alertrules"
-        exact
-        loader={() =>
-          import('./monitoring/alerting' /* webpackChunkName: "alerting" */).then(
-            (m) => m.MonitoringUI,
-          )
-        }
-      />
-      <LazyRoute
-        path="/monitoring/silences"
-        exact
-        loader={() =>
-          import('./monitoring/alerting' /* webpackChunkName: "alerting" */).then(
-            (m) => m.MonitoringUI,
-          )
-        }
-      />
-      <LazyRoute
-        path="/monitoring/alertmanageryaml"
-        exact
-        loader={() =>
-          import('./monitoring/alerting' /* webpackChunkName: "alerting" */).then(
-            (m) => m.MonitoringUI,
-          )
-        }
-      />
-      <LazyRoute
-        path="/monitoring/alertmanagerconfig"
-        exact
-        loader={() =>
-          import('./monitoring/alerting' /* webpackChunkName: "alerting" */).then(
-            (m) => m.MonitoringUI,
-          )
-        }
-      />
-      <LazyRoute
-        path="/monitoring/alertmanagerconfig/receivers/~new"
-        exact
-        loader={() =>
-          import(
-            './monitoring/receiver-forms/alert-manager-receiver-forms' /* webpackChunkName: "receiver-forms" */
-          ).then((m) => m.CreateReceiver)
-        }
-      />
-      <LazyRoute
-        path="/monitoring/alertmanagerconfig/receivers/:name/edit"
-        exact
-        loader={() =>
-          import(
-            './monitoring/receiver-forms/alert-manager-receiver-forms' /* webpackChunkName: "receiver-forms" */
-          ).then((m) => m.EditReceiver)
-        }
-      />
```

Comment:
The `alertmanagerconfig` and `alertmanageryaml` pages are not part of the plugin, so these routes should not be removed.